index.css

```diff
@@ -1,8 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@1,300&display=swap');
+
 body {
   max-width: 375px;
-  font-family: Poppins, sans-serif;
-  margin: 0;
-  background-color: #e5e5e5;
+  background-color: white;
+  height: fit-content;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  box-shadow: 5px 10px;
 }
 
 .header {
```

Comment:
- [ ] Good job implementing the CSS Flexbox in the `header` class. I suggest you change the `header` class to the `navbar` class to follow HTML semantic tag requirements. 😄